src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -1069,6 +1069,28 @@
     </Environment>
   </Provider>
 
+  <!--
+  LINEAR
```

Comment:
How do I create the text graphic for the names?

Reply:
You can use https://textfancy.com/text-art/ with the font `Small Negative` (make sure you add a space after the provider name so it doesn't look weird). I'll update the docs to add a link.

Reply:
> I'll update the docs to add a link.

Done: https://documentation.openiddict.com/guides/contributing-a-new-web-provider#add-the-ascii-art-header.